<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewBookingDialog</class>
 <widget class="QDialog" name="NewBookingDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>240</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>221</width>
     <height>230</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QDoubleSpinBox" name="input_price">
      <property name="suffix">
       <string> Euro</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QSpinBox" name="reise_id_input"/>
    </item>
    <item>
     <widget class="QDateEdit" name="fromDate_input"/>
    </item>
    <item>
     <widget class="QDateEdit" name="to_date_input"/>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <item>
       <property name="text">
        <string>Flugbuchung</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Hotelbuchung</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Autoreservierung</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="change_box1"/>
    </item>
    <item>
     <widget class="QLineEdit" name="change_box2"/>
    </item>
    <item>
     <widget class="QLineEdit" name="change_box3"/>
    </item>
    <item>
     <widget class="QLineEdit" name="change_box3_2"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>141</width>
     <height>231</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Preis</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Reise-ID</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>Startdatum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Enddatum</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>Buchungstyp</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="change_label1">
      <property name="text">
       <string>Startflughafen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="change_label2">
      <property name="text">
       <string>Zielflughafen</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="change_label3">
      <property name="text">
       <string>Fluglinie</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="change_label4">
      <property name="text">
       <string>Sitzpr√§ferenz</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>NewBookingDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>NewBookingDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
